doc/lua_api.txt

```diff
@@ -5317,22 +5317,26 @@ Call these functions only at load time!
 * `minetest.register_craft_predict(function(itemstack, player, old_craft_grid, craft_inv))`
     * The same as before, except that it is called before the player crafts, to
       make craft prediction, and it should not change anything.
-* `minetest.register_allow_player_inventory_action(function(player, action, inventory, inventory_info))`
-    * Determines how much of a stack may be taken, put or moved to a
-      player inventory.
+* `minetest.register_on_player_inventory_action(function(player, action, inventory, inventory_info))`
+    * Called after an item take, put or move event from/to/in a player inventory
+    * These inventory actions are recognized:
+        * move: Item was moved within the player inventory
+        * put: Item was put into player inventory from another inventory
+        * take: Item was taken from player inventory and put into another inventory
     * `player` (type `ObjectRef`) is the player who modified the inventory
       `inventory` (type `InvRef`).
     * List of possible `action` (string) values and their
       `inventory_info` (table) contents:
         * `move`: `{from_list=string, to_list=string, from_index=number, to_index=number, count=number}`
         * `put`:  `{listname=string, index=number, stack=ItemStack}`
         * `take`: Same as `put`
+    * Does not accept or handle any return value.
+* `minetest.register_allow_player_inventory_action(function(player, action, inventory, inventory_info))`
+    * Determines how much of a stack may be taken, put or moved to a
+      player inventory.
+    * Function arguments: see `minetest.register_on_player_inventory_action`
     * Return a numeric value to limit the amount of items to be taken, put or
       moved. A value of `-1` for `take` will make the source stack infinite.
```

Comment:
This is called before the `register_on_player_inventory_action`, right?
Does returning 0 prevent the `register_on_player_inventory_action` callback? If yes, please add a note.